### ISSUE TITLE
Data Analyst able to confirm full Northstar dataset exists in Quasar Users Table

### ISSUE THREAD

**Issue by jessleenyc** [DoSomething/quasar#294]
_TASK_
-------------------
#### Description
As a Data Analyst, I want to review user data in the Quasar `Users` table so that we can confirm that all Northstar user data has been ingested into the Quasar `Users` table. 

#### Why This Matters
Reviewing the Northstar dataset ingestion is necessary so that we are confident that all user information exists in the Quasar `Users` table. 

#### Definition of Done
-The following checklist is completed: https://docs.google.com/document/d/1xKsmhPamo1RqwofqHsi0fv5O3F-YoPFhQrVS7yCKIx0/edit#

**Comment by sheyd** [comment]
Quasar ingestion is still on-going. Looks like it should be complete by Monday!

**Comment by sheyd** [comment]
User count matches Northstar count! Let the reviews begin!

**Comment by Bladt** [comment]
Reviewing has started. 

Here is a Look for those playing along at home to see Users by NS Create Date and NS Source: https://data.dosomething.org/looks/446?toggle=det,fil

**Comment by Bladt** [comment]
I tried to group by first name. Got this error. Something might be going on with indexing:
![image](https://cloud.githubusercontent.com/assets/4752626/24169735/f30de7a8-0e54-11e7-903f-dcfe5bdb962f.png)

**Comment by nmccormick619** [comment]
What's the deal with the Drupal uptick on 2/08?
<img width="1143" alt="screen shot 2017-03-21 at 5 22 29 pm" src="https://cloud.githubusercontent.com/assets/17914600/24171471/0063fa68-0e5b-11e7-9ca6-0815bd6a6c0b.png">

**Comment by sheyd** [comment]
I've put in a PR ^^ to update user count. There are some comments in there I want to get into this PR that will help things considerably. Also investigating a slowdown in the Northstar scraping again. I'll update in another ticket.

**Comment by Bladt** [comment]
I need to update mgmt-team on when we will be able to run current member count. When do we think we will have rows 24-38 (pre-reqs for a member count) from the schema?

**Comment by jamjensen** [comment]
Blocked by #321

**Comment by Bladt** [comment]
using the following to test member count:

'
select 
  count(case when qu.mailchimp_subscription_status = 'subscribed' or qu.moco_current_status = 'subscribed' then qu.northstar_id end) as 'total_members',
  count(case when qu.mailchimp_subscription_status = 'subscribed' then qu.northstar_id end) as 'email_subs',
  count(case when qu.moco_current_status = 'subscribed' then qu.northstar_id end) as 'sms_subs'
from quasar.users qu
'

**Comment by Bladt** [comment]
updated:
```
select 
  count(case when qu.mailchimp_subscription_status  is not null or qu.moco_current_status is not null then qu.northstar_id end) as 'all_time_members',
  count(case when qu.mailchimp_subscription_status = 'subscribed' or qu.moco_current_status = 'active' then qu.northstar_id end) as 'total__active_members',
  count(case when qu.mailchimp_subscription_status = 'subscribed' then qu.northstar_id end) as 'email_subs',
  count(case when qu.moco_current_status = 'active' then qu.northstar_id end) as 'sms_subs'
from quasar.users qu
```

**Comment by jamjensen** [comment]
When looking up members that live in Georgia, we noticed a discrepancy between what we saw in Hubble compared to that in Quasar. To confirm that all necessary information is moving over to Quasar, we need to QA: 

- [ ] State
- [ ] Mobile numbers
- [ ] Zip code
- [ ] All first names
- [ ] Birthdates 
- [ ] Gender

**Comment by jamjensen** [comment]
**State** 
- There are 5392 members in Quasar with `State` = GA and `mailchimp status` = 'subscribed.' 
- There are 3368 members who are in Quasar with `mailchimp status` = 'subscribed,' with `State` = null AND with `hubble state` = 'GA.'  We are missing these users' values for `State` in Quasar.
Query: 
```select df.field_address_administrative_area as 'hubble state', df.entity_id as 'uid', u.mailchimp_subscription_status, u.northstar_id, u.email, u.addr_state
from dosomething.field_data_field_address df
left join quasar.users u
	ON df.entity_id =u.drupal_uid
where df.field_address_administrative_area = 'GA'
AND mailchimp_subscription_status = 'subscribed'
AND u.addr_state IS NULL
```
- Link to user's profile in Northstar when we have a value for `state` in Quasar: https://aurora.dosomething.org/users/55a82463469c64022e8ed5f0 
- Link to user's profile in Northstar when we do not have a value for `state` in Quasar but do have a value for `state` in Hubble/phoenix: 
https://aurora.dosomething.org/users/55a7546d469c649e2b8ce527

**Comment by DFurnes** [comment]
Yeah, it looks like the second user's address wasn't successfully synced into Northstar. I'm not really sure why that would be the case though...

**Comment by jamjensen** [comment]
Update from when Bladt last ran this query on March 21: there are now 4,480 drupal IDs shared by 2 or more Northstar IDs. 
``` 
SELECT 
	quasar_users.drupal_uid  AS `quasar_users.drupal_uid`,
	COUNT(*) AS `quasar_users.count`
FROM quasar.users  AS quasar_users

GROUP BY 1
HAVING 
	(COUNT(*) >1)
ORDER BY COUNT(*) DESC
LIMIT 5000
```
